pumps/sql_aggregate_test.go

```diff
@@ -191,7 +191,7 @@ func TestSQLAggregateWriteDataValues(t *testing.T) {
 		t.Fatal("SQL Pump Aggregate couldn't be initialized with err: ", err)
 	}
 	defer func(table string) {
-		pmp.db.Migrator().DropTable(analytics.AggregateSQLTable)
+		//pmp.db.Migrator().DropTable(analytics.AggregateSQLTable)
```

Comment:
should we remove entire defer function?

Reply:
I think it needs be uncommented, since its goal is to clean up after the test.

Reply:
thanks!